### DISCUSSION TITLE
Not able to run media from asset

### DISCUSSION THREAD

**Discussion by yogesharora388** [wang-bin/fvp#46] [General]
I am trying to run the media using assets://assets/1.mov path but player is not loading the media.
I am using the player API I.e player.media

**Comment by wang-bin** [comment]
which platform? show me the log

**Comment by wang-bin** [comment]
it's in video_player implementation:
https://github.com/wang-bin/fvp/blob/v0.13.0/lib/src/video_player_mdk.dart#L320
